packages/helm-charts/blockscout/values-alfajores-blockscout2.yaml

```diff
@@ -30,16 +40,41 @@ blockscout:
       proxy:
         resources:
           requests:
-            memory: 20M
-            cpu: 10m
+            memory: 10M
+            cpu: 25m
     poolSize: 25
     resources:
       requests:
-        memory: 600M
+        memory: 300M
         cpu: 0.5
       limits:
-        memory: 1200M
+        memory: 800M
         cpu: 1.2
+    affinity:
+      nodeAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            preference:
+              matchExpressions:
+                - key: node.kubernetes.io/instance-type
+                  operator: In
+                  values:
+                    - n2-highmem-2
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 99
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: release
+                    operator: In
+                    values:
+                      - alfajores-blockscout2
+                  - key: component
+                    operator: In
+                    values:
+                      - blockscout-api
```

Comment:
Why do both api and web pods have the same anti-affinity rule?

Reply:
Good seen! Thanks